libs/ui/src/hooks/smartcard_auth/use_dipped_smartcard_auth.ts

```diff
@@ -302,18 +281,12 @@ async function logAuthEvents(
   previousAuth: DippedSmartcardAuth.Auth = {
     status: 'logged_out',
     reason: 'machine_locked',
-    bootstrapAuthenticatedElectionManagerSession:
-      /* istanbul ignore next */ () => undefined,
   },
   auth: DippedSmartcardAuth.Auth
 ) {
   switch (previousAuth.status) {
     case 'logged_out': {
-      if (auth.status === 'logged_in') {
-        await logger.log(LogEventId.AuthLogin, auth.user.role, {
-          disposition: LogDispositionStandardTypes.Success,
-        });
-      } else if (
```

Comment:
This transition from `logged_out` straight to `logged_in` was only possible with a call to `bootstrapAuthenticatedElectionManagerSession`. With `bootstrapAuthenticatedElectionManagerSession` removed, this case is no longer needed.

More on why we can remove `bootstrapAuthenticatedElectionManagerSession` in https://github.com/votingworks/vxsuite/pull/2025#discussion_r908563253 and why we can remove this case in https://github.com/votingworks/vxsuite/pull/2036#discussion_r913816744